.github/CODEOWNERS

```diff
@@ -1,8 +1,18 @@
 * @FabricMC/documentation
 
+.gitattributes @FabricMC/developers
```

Comment:
I think we should changed this file to default to devs, but allow the docs team access to the content.